nsiqunittest/nsiqcppstyle_unittest.py

```diff
@@ -27,7 +27,10 @@
 
 import unittest
 import nsiqcppstyle_checker
+from nsiqcppstyle_outputer import _consoleOutputer as _consoleOutputer

```

Comment:
Let's keep the import name uniform and call it a `console` instead of `_consoleOutputer`